3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/CurrencyConversionEnrichmentSpec.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2012-2014 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.enrich.common
+package enrichments
+package registry
+
+// Specs2
+import org.specs2.Specification
+import org.specs2.matcher.DataTables
+import org.specs2.scalaz._
+
+// Scalaz
+import scalaz._
+import Scalaz._
+
+
+// Scala Forex
+import org.joda.time.{DateTime, DateTimeZone}
+import com.snowplowanalytics.forex._
+import com.snowplowanalytics.forex.oerclient.{OerClientConfig, DeveloperAccount, AccountType}
+import com.snowplowanalytics.forex.{Forex, ForexConfig}
+
+import com.snowplowanalytics.snowplow.enrich.common.enrichments.registry._
+// This project
+//import object com.snowplowanalytics.snowplow.enrich.common.enrichments.registry.CurrencyConversionEnrichment
+/**
+ * Tests the convertCurrencies function
+ */
+
+class CurrencyConversionEnrichmentSpec extends Specification with DataTables { def is =
+
+  "This is a specification to test convertCurrencies"                  ^
+                                                                      p^
+    "Currency Conversion should work"                               ! e1^
+                                                                      end
+
+  val appIdInvalid = (TransactionAmounts("Open Exchange Rates error, message: Invalid App ID provided - please sign up at https://openexchangerates.org/signup, or contact support@openexchangerates.org. Thanks!","Open Exchange Rates error, message: Invalid App ID provided - please sign up at https://openexchangerates.org/signup, or contact support@openexchangerates.org. Thanks!","Open Exchange Rates error, message: Invalid App ID provided - please sign up at https://openexchangerates.org/signup, or contact support@openexchangerates.org. Thanks!"),"Open Exchange Rates error, message: Invalid App ID provided - please sign up at https://openexchangerates.org/signup, or contact support@openexchangerates.org. Thanks!")
```

Comment:
Why would these errors be returned inside a TransactionAmounts? They are error messages, not successful double-like Strings.